### PR TITLE
fix: use div instead of DisclosureButton for decls in the module tree

### PR DESCRIPTION
**Pull request by deniseli**
This fixes a `Cannot read properties of undefined (reading 'length')` error that was appearing every time you went to the modules page, which was coming from inside `DisclosureButton`. The error's stack trace was minified beyond comprehension, but I'm guessing it was because this was a malformed usage of `DisclosureButton` - not wrapped in a corresponding `Disclosure` (its parent `Disclosure` component mapped to the higher level module buttons, not the decl items).

### CODE REVIEW

**Review by wesbillman — APPROVED** [review]

**Review by gak — APPROVED** [review]

File: frontend/console/playwright-report/index.html [hunk 1]
```diff
@@ -65,4 +65,4 @@
...
...Aaejo+ggYs7YUQMEqrkE7bkLUEo17Ub9+3X7uF5deZmr/zRCnWtyru7f4KCLrERQJAh2lDL6KJai5iK6K/1ZCwnvxLgjFiOAcONSeC6EdQaC5hK6p/1ZAprxLB/3mIQDEYxBeVzxC6QSWurocAts2Ds2U+EesLIM1iOAVBStMDDpIKV/V7Ym60C+qtltE2jft15z5zBHwKsCJull3n8IVk4DQVlYSCiVRKQ6afLgN0NUBMJeP8Nq+Cz6RvCXnhQgSPUofnRS38bkm/7l4nPgfvPtRcYzKk0BnK19VpjQwJvuJc/+YmSWRUVJYoYHnaoZSqtfzGjDVT7Q4e7lJ8XMOtF7Je7f0KBuFivtYklGOq8iVk1bPZfMTYp+BRnLP3wUaK6IjLagCzSNZI/zINJxAM63zGWSsvsv55/cKXZssP2G1i1Ry7SlKfSn0PEAnvu5HbCwl6KC9A+0DcR6jM4qPDuc8m7czBf6vmE6FynP3zQ/6ux9VR6GAVloooUkHFCC0Qzd+5BmHclXZM5lIdf+3+CgTlGARS4/EuSjRG8XtDCbXJD0TiePuvynoxwso2c9L/fzomjOrkQBE+TKCvJsXQsGo69rulOI/UEsBAj8DFAAACAgA6IIjWc7yDTLuAwAAmBwAABkAAAAAAAAAAAAAALSBAAAAADQyYjY3N2ViYTU2NGE5MGRiYjE4Lmpzb25QSwECPwMUAAAICADogiNZ42Ur7WQEAAA9JAAAGQAAAAAAAAAAAAAAtIElBAAAYTU1NzFmYjM3MmMwYzkxYjZlOGIuanNvblBLAQI/AxQAAAgIAOiCI1l2DDWwpgQAACYjAAAZAAAAAAAAAAAAAAC0gcAIAABhYWUwZGIxOTRmMGRhNTgxYzU0Zi5qc29uUEsBAj8DFAAACAgA6IIjWSD3w/lcBAAAhyAAABkAAAAAAAAAAAAAALSBnQ0AAGMzMGNkMzllMWM4N2Y5ZmY2NTZhLmpzb25QSwECPwMUAAAICADogiNZtmDNlkkFAACCLAAAGQAAAAAAAAAAAAAAtIEwEgAANDY1MjNlMTc0OGRlNTA0OWVkMjQuanNvblBLAQI/AxQAAAgIAOiCI1l97UzPbwMAAKAUAAALAAAAAAAAAAAAAAC0gbAXAAByZXBvcnQuanNvblBLBQYAAAAABgAGAJwBAABIGwAAAAA=";</script>
```
Comment (wesbillman) [inline frontend/console/playwright-report/index.html:68]: I don't think this file should be in the repo at all 🤔

Comment (wesbillman) [reply frontend/console/playwright-report/index.html:68]: I'll fix in a follow on PR

Comment (deniseli) [reply frontend/console/playwright-report/index.html:68]: Whoops thanks for catching that! I copied your gitignore change from adding the infra page.

Comment (deniseli) [reply frontend/console/playwright-report/index.html:68]: Whoops x2, I accidentally pushed a fix for this before seeing your second comment. I'll slack you

**Pull request merged**